geoh5py/objects/curve.py

```diff
@@ -26,17 +26,74 @@
 from .points import Points
 
 
-class Curve(Points):
+class CellObject(Points):
+    """
+    Base class for object with cells.
+    """
+
+    _attribute_map = Points._attribute_map.copy()
+
+    def __init__(self, object_type: ObjectType, **kwargs):
+
+        self._cells: np.ndarray | None = None
+
+        super().__init__(object_type, **kwargs)
```

Comment:
**suggestion:** usually, calling parent constructor comes first

Reply:
Ok maybe, but the base `Entity` applies the kwargs, so `_cells` would be lost.